index.html

```diff
@@ -44,8 +48,167 @@ <h1 class="prmrytxt1">
+        <li class="tgs">css</li>
+        <li class="tgs">javascript</li>
+      </ul>
+      <div >
+        <button class="prjctbtn">
+          <span class="btntxt">See Project</span>
+        </button>
+      </div>
+    </div>
+    <div class="port1">
+      <img src="images/card2pic.png" alt="Tonic">
+      <div>
+        <h2 class="title">Multi-Post <br> Stories</h2>
+        <div>
+          <span class="clnt">CANOPY</span>
+          <img src="images/circleicn.png" alt="Circle" class="crcl">
+          <span class="deets">Back End Dev</span>
+          <img src="images/circleicn.png" alt="Circle" class="crcl">
+          <span class="deets">2015</span>
+        </div>
+      </div>
+      <p>
+        A daily selection of privately personalized reads; no accounts or sign-ups required.
+      </p>
+      <ul class="tags">
+        <li class="tgs">html</li>
+        <li class="tgs">css</li>
+        <li class="tgs">javascript</li>
+      </ul>
+      <div>
+        <button class="prjctbtn">
+          <span class="btntxt">See Project</span>
+        </button>
+      </div>
+    </div>
+    <div class="port1">
+      <img src="images/card3pic.png" alt="Tonic">
+      <div>
+        <h2 class="title">Tonic</h2>
+        <div>
+          <span class="clnt">CANOPY</span>
+          <img src="images/circleicn.png" alt="Circle" class="crcl">
+          <span class="deets">Back End Dev</span>
+          <img src="images/circleicn.png" alt="Circle" class="crcl">
+          <span class="deets">2015</span>
+        </div>
+      </div>
+      <p>
+        A daily selection of privately personalized reads; no accounts or sign-ups required.
+      </p>
+      <ul class="tags">
+        <li class="tgs">html</li>
+        <li class="tgs">css</li>
+        <li class="tgs">javascript</li>
+      </ul>
+      <div>
+        <button class="prjctbtn">
+          <span class="btntxt">See Project</span>
+        </button>
+      </div>
+    </div>    
+    <div class="port1">
+      <img src="images/card4pic.png" alt="Tonic">
+      <div>
+        <h2 class="title">Multi-Post <br> Stories</h2>
+        <div>
+          <span class="clnt">CANOPY</span>
+          <img src="images/circleicn.png" alt="Circle" class="crcl">
+          <span class="deets">Back End Dev</span>
+          <img src="images/circleicn.png" alt="Circle" class="crcl">
+          <span class="deets">2015</span>
+        </div>
+      </div>
+      <p >
+        A daily selection of privately personalized reads; no accounts or sign-ups required.
+      </p>
+      <ul class="tags">
+        <li class="tgs">html</li>
+        <li class="tgs">css</li>
+        <li class="tgs">javascript</li>
+      </ul>
+      <div>
+        <button class="prjctbtn">
+          <span class="btntxt">See Project</span>
+        </button>
+      </div>
+    </div>
+  </section>
+
+  <!---About Section-->
+  <section class="about">
+    <i class="prymrytxt1">
+      About <br> Myself
+    </i>
```

Comment:
Well done on this project 🥳 the UI, the PR, everything looks great but to talk semantics a bit, you could improve this by making sure that the title uses a header tag `hx`. That will make it easy for people to understand your web app and for browsers 🌐 as well.